### ISSUE TITLE
packaged pg-activity not running on ubuntu 20.04

### ISSUE THREAD

**Issue by b0n0r** [dalibo/pg_activity#325]
installed from Postgresql.Org repo:

```
2022-11-14 09:25:16 status installed python3-wcwidth:all 0.1.8+dfsg1-3
2022-11-14 09:25:17 status installed python3-psycopg2:amd64 2.8.6-2~pgdg20.04+1
2022-11-14 09:25:17 status installed python3-humanize:all 1.0.0-1
2022-11-14 09:25:17 status installed python3-blessed:all 1.16.1-1
2022-11-14 09:25:17 status installed pg-activity:all 3.0.1-1.pgdg20.04+1
2022-11-14 09:25:18 status installed man-db:amd64 2.9.1-1
```

package is installed:

`ii  pg-activity                           3.0.1-1.pgdg20.04+1               all          Realtime PostgreSQL database server monitoring tool`

however is unable to start:

```
$ pg_activity --help
Traceback (most recent call last):
  File "/usr/bin/pg_activity", line 11, in <module>
    load_entry_point('pg-activity==3.0.1', 'console_scripts', 'pg_activity')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2854, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3/dist-packages/pgactivity/cli.py", line 13, in <module>
    from . import __version__, data, types, ui
  File "/usr/lib/python3/dist-packages/pgactivity/data.py", line 24, in <module>
    from .types import (
  File "/usr/lib/python3/dist-packages/pgactivity/types.py", line 25, in <module>
    from . import compat, colors, utils
  File "/usr/lib/python3/dist-packages/pgactivity/colors.py", line 1, in <module>
    from . import utils
  File "/usr/lib/python3/dist-packages/pgactivity/utils.py", line 12, in <module>
    humanize.precisedelta,
AttributeError: module 'humanize' has no attribute 'precisedelta'
```

there is nothing specific or exotic about the environment, simple box that runs postgresql and sshd, that's about it.

**Comment by dlax** [comment]
That's a problem in the [Debian packaging](https://www.postgresql.org/message-id/CAGs4muVPttuu1Mw2awQ%3DLR3XxC68es9jbc4x6_%2Bi5FFgQe79XA%40mail.gmail.com), nothing we can do on our side.

**Comment by deric** [comment]
@b0n0r A workaround is to update `humanize` package, which is outdated in [Debian Buster](https://packages.debian.org/buster/python3-humanize) (and derived distributions):
```
python3 -m pip install -U humanize
```

**Comment by lanaparadinha** [comment]
postgres@DCCBDL01:~$ python3 -m pip install -U humanize
/usr/bin/python3: No module named pip

**Comment by dlax** [comment]
> psycopg2.errors.SyntaxError: syntax error at or near ""
> LINE 16: convert_from(replace(a.query, '', '\')::bytea, coale...

This particular problem got resolved in a later release (3.1.0) of pg_activity. Unfortunately, it's not available in Debian/Ubuntu (yet).
